<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PeriodicTableWidget</class>
 <widget class="QWidget" name="PeriodicTableWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>489</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QListWidget" name="pteList"/>
   </item>
   <item>
    <widget class="QWidget" name="pteContainer" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="13" column="0" colspan="3">
       <widget class="QPushButton" name="toGlobalButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Save element to global table</string>
        </property>
       </widget>
      </item>
      <item row="11" column="0" colspan="3">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="covLabel">
        <property name="text">
         <string>Co&amp;valent radius:</string>
        </property>
        <property name="buddy">
         <cstring>covSel</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="mLabel">
        <property name="text">
         <string>Mass:</string>
        </property>
        <property name="buddy">
         <cstring>mSel</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QSpinBox" name="zSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="maximum">
         <number>200</number>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QPushButton" name="colSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Select</string>
        </property>
       </widget>
      </item>
      <item row="12" column="0" colspan="3">
       <widget class="QPushButton" name="newElemButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Create/replace element</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="vdwSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="covSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="colLabel">
        <property name="text">
         <string>Display color:</string>
        </property>
        <property name="buddy">
         <cstring>colSel</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="cutSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="pwppLabel">
        <property name="text">
         <string>PWScf PP-File:</string>
        </property>
        <property name="buddy">
         <cstring>pwppSel</cstring>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="pwppSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="10" column="1" colspan="2">
       <widget class="QLineEdit" name="cpnlSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="cpppSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="cutLabel">
        <property name="text">
         <string>Bo&amp;nd cutoff radius:</string>
        </property>
        <property name="buddy">
         <cstring>cutSel</cstring>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="3">
       <widget class="QPushButton" name="fromGlobalButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Load element from global table</string>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="cpnlLabel">
        <property name="text">
         <string>CPMD Nonlocality:</string>
        </property>
        <property name="buddy">
         <cstring>cpnlSel</cstring>
        </property>
       </widget>
      </item>
      <item row="18" column="0" colspan="3">
       <widget class="QPushButton" name="defaultButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Reset to element defaults</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="vdwLabel">
        <property name="text">
         <string>Van-der-Waals radius:</string>
        </property>
        <property name="buddy">
         <cstring>vdwSel</cstring>
        </property>
       </widget>
      </item>
      <item row="6" column="1" colspan="2">
       <widget class="QDoubleSpinBox" name="mSel">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="decimals">
         <number>4</number>
        </property>
        <property name="maximum">
         <double>1000.000000000000000</double>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="cpppLabel">
        <property name="text">
         <string>CPMD PP-File:</string>
        </property>
        <property name="buddy">
         <cstring>cpppSel</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="zLabel">
        <property name="text">
         <string>Ato&amp;mic number:</string>
        </property>
        <property name="buddy">
         <cstring>zSel</cstring>
        </property>
       </widget>
      </item>
      <item row="19" column="0" colspan="3">
       <widget class="QPushButton" name="deleteButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Delete custom element</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="helpButton">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>25</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string>?</string>
        </property>
       </widget>
      </item>
      <item row="15" column="0" colspan="3">
       <widget class="QPushButton" name="cleanButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="text">
         <string>Remove unused elements</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>setEntry(QListWidgetItem*)</slot>
  <slot>changeEntry()</slot>
 </slots>
</ui>
